src/inspect_evals/swe_bench/swe_bench.py

```diff
@@ -61,6 +62,10 @@ def swe_bench(
     """
     getLogger().handlers = []  # Swe-bench adds a global logger, which we disable.
 
+    assert find_spec(
+        "swebench"
```

Comment:
This is fine (although I think the optional dependency is named `swe_bench`?)

Reply:
Yup, will change!